src/ptwt/_stationary_transform.py

```diff
@@ -28,7 +29,7 @@ def _swt(
         level (Optional[int], optional): The number of levels to compute
 
     Returns:
-        List[torch.Tensor]: Same as wavedec.
+        list[torch.Tensor]: Same as wavedec.
```

Comment:
as you go through these, it's better to remove the redundant type annotations inside the docstrings rather than update them

```suggestion
        : Same as wavedec.
````

Reply:
Should we keep the leading colons? The examples in [Google's styleguide](https://google.github.io/styleguide/pyguide.html#doc-function-returns) show none.

Reply:
if the documentation works without it, then no! I am not usually using this code style so I don't know all the specifics

Reply:
I went through the docstrings and cleaned them up. This should be addressed now.

The styleguide also writes about function arguments: "The description should include required type(s) if the code does not contain a corresponding type annotation." Opinions on this matter?